crates/frontend/src/types.rs

```diff
@@ -314,3 +314,19 @@ pub struct FetchTypeTemplateResponse {
     pub total_pages: i64,
     pub data: Vec<TypeTemplate>,
 }
+
+#[derive(Debug, Clone, Deserialize, Serialize)]
+pub struct SnapshotResponse {
```

Comment:
You may have to change it similar to what i have done for the endpoint response type.

Reply:
Use PaginatedResponse from superposition_types

Reply:
We don't have superposition_types in frontend , Let's move all the common types in a seperate PR. 
What do you suggest?

Reply:
@sauraww then change it to snapshotlistresponse